docs/standard/whats-new/whats-new-in-dotnet-standard.md

```diff
@@ -0,0 +1,87 @@
+
+Some of the more noteworthy additions to the <xref:System> namespace alone in .NET Standard 2.0 include the following:
+
+- Support for the <xref:System.AppDomain>] class.
+- Better support for working with arrays from additional members in the <xref:System.Array> class.
+- Better support for working with attributes from additional members in the <xref:System.Attribute> class.
+- Better calendar support and additional formatting options for <xref:System.DateTime> values.
+- Additional <xref:System.Decimal> rounding functionality.
+- Additional functionality in the <xref:System.Environment> class.
+- Enhanced control over the garbage collector through the <xref:System.GC> class.
+- Enhanced support for string comparison, enumeration, and normalization in the <xref:System.String> class.
+- Support for daylight saving adjustments and transition times in the <xref:System.TimeZoneInfo.AdjustmentRule> and <xref:System.TimeZoneInfo.TransitionTime> classes.
+- Significantly enhanced functionality in the <xref:System.Type> class.
+- Better support for deserialization of exception objects by adding an exception constructor with <xref:System.Runtime.Serialization.SerializationInfo> and <xref:System.Runtime.Serialization.StreamingContext> parameters.
+
+**Support for .NET Framework libraries**
+
+The overwhelming majority of libraries target the .NET Framework rather than .NET Standard. However, most of the calls in those libraries are to APIs that are included in the .NET Standard 2.0. Starting with the .NET Standard 2.0, you can access .NET Framework libraries from a .NET Standard library by using a [compatibility shim](https://github.com/dotnet/standard/blob/master/docs/netstandard-20/README.md#assembly-unification). This compatibility layer is transparent to developers; you don't have to do anything to take advantage of .NET Framework libraries.
+
+The single requirement is that the APIs called by the .NET Framework class library must be included in the .NET Standard 2.0.
+
+**Support for Visual Basic**
+
+You can now develop .NET Standard libraries in Visual Basic. For Visual Basic developers using Visual Studio 2017, Visual Studio now includes a .NET Standard Class Library template. For Visual Basic developers who use other development tools and environments, you can use the [dotnet new](../../core/tools/dotnet-new.md) command to create a .NET Standard Library project. For more information, see the [Tooling support for .NET Standard libraries](#tooling).
```

Comment:
Is there a minimum version of Visual Studio needed for the .NET Standard Template?